frame/include/bli_type_defs.h

```diff
@@ -1006,20 +1006,29 @@ typedef enum
 	BLIS_ARCH_CORTEXA53,
 	BLIS_ARCH_CORTEXA15,
 	BLIS_ARCH_CORTEXA9,
+	BLIS_ARCH_A64FX,
 
 	// IBM/Power
 	BLIS_ARCH_POWER9,
 	BLIS_ARCH_POWER8,
 	BLIS_ARCH_POWER7,
 	BLIS_ARCH_BGQ,
 
+    // IBM S390
+	BLIS_ARCH_Z13,
+	BLIS_ARCH_Z14,
+
 	// Generic architecture/configuration
 	BLIS_ARCH_GENERIC,
 
   // Number of architectures, must be last
   BLIS_NUM_ARCHS
 } arch_t;
 
+// NOTE: This value must be updated to reflect the number of enum values
+// listed above for arch_t!
+#define BLIS_NUM_ARCHS 25
+
```

Comment:
This isn't needed anymore, see last entry in `arch_t`.

Reply:
Can you explain this comment, Devin?

Reply:
In another PR, I made BLIS_NUM_ARCHS a member of the arch_t enum (the last one) so that it will always be up-to-date.

Reply:
Gotcha. I hadn't seen that yet.